package.json

```diff
@@ -12,6 +12,13 @@
     "container"
   ],
   "main": "lib/preload.js",
+  "files": [
+    "bin/resin-preload",
+    "lib/preload.js",
+    "src/preload.py",
+    "Dockerfile",
+    "requirements.txt"
+  ],
```

Comment:
Instead of whitelisting things explicitly, I think it'd be less error prone to just add things like `__pycache__` to the `.npmignore`